src/test/java/com/neuvector/ScannerTest.java

```diff
@@ -41,7 +42,7 @@ public void scanLocalImageTest() throws IOException {
         String nvRegistryPassword = null;
         String nvRegistryURL = "https://registry.hub.docker.com";
         //mountPath is an optional parameter. It will use "/var/neuvector" by default.
-        String mountPath = mountFolder.getRoot().getAbsolutePath();
+        String mountPath = mountFolder.getAbsolutePath();
```

Comment:
Reason for the change? Looks unrelated to the task at hand?

Reply:
Upgraded jUnit to v5 as part of this.